lib/linguist/languages.yml

```diff
@@ -20,7 +20,7 @@
 # language_id           - Integer used as a language-name-independent indexed field so that we can rename
 #                         languages in Linguist without reindexing all the code on GitHub. Must not be
 #                         changed for existing languages without the explicit permission of GitHub staff.
-# color                 - CSS hex color to represent the language. Only used if type is "programming" or "markup".
+# color                 - CSS hex color to represent the language. Only used if type is "programming" or "prose".
```

Comment:
Looks like this is reverting #4247 by mistake.

Reply:
@KrzysztofSzewczyk Could you please do a fresh pull on your branch, making sure that all recent changes this week are in your branch?

Reply:
@Alhadis how do I make a fresh pull?

Reply:
The [SO answer](https://stackoverflow.com/a/3877000/6884590) I've linked to before should work fine for that.